projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/src/hooks/useSocialLogin.tsx

```diff
@@ -32,15 +43,32 @@ export default function useSocialLogin() {
 	const [ loginWindowRef, setLoginWindowRef ] = useState< Window >();
 
 	useEffect( () => {
-		wpcomRequest< UserInfo >( {
-			path: '/verbum/auth',
-			apiNamespace: 'wpcom/v2',
-		} ).then( res => {
-			userInfo.value = res;
+		if ( ! ( 'hasStorageAccess' in document ) ) {
+			canAccessCookies.value = canWeAccessCookies();
```

Comment:
The `canWeAccessCookies` function should also be updated, to make it more reliable. Namely, set the `verbum_test` with the `Partitioned` attribute. Otherwise browsers will ignore it when the Verbum iframe is embedded in a non-wordpress.com domain.